proto/buf.gen.yaml

```diff
@@ -5,8 +5,8 @@ managed:
     default: "github.com/conduit-labs/conduit-connector-grpc-client/proto"
     except:
       - buf.build/googleapis/googleapis
-    override:
-      buf.build/conduitio/conduit-connector-protocol: github.com/conduitio/conduit-connector-protocol/proto
+    override:                                         
+      buf.build/conduitio/conduit-connector-protocol: github.com/conduitio/conduit-commons/proto
```

Comment:
Hmm, why not use `buf.build/conduitio/conduit-commons`?

Reply:
yes sorry, fixed now!